cloud_functions/shared/cloud_utils.py

```
@@ -5,26 +5,28 @@
     if "PROJECT_ID" not in environ:
         raise KeyError("PROJECT_ID environment variable not found")
-    logger.debug("Initializing secret manager")
+    partial_log_structured(message="Initializing secret manager")
     client = secretmanager.SecretManagerServiceClient()
-    logger.debug(f"Getting secret for {name}")
+    partial_log_structured(message=f"Getting secret for {name}")
     try:
         response = client.access_secret_version(
             request={
                 "name": f"projects/{environ['PROJECT_ID']}/secrets/{name}/versions/latest",
             }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)